lib/gds_api/publishing_api_v2.rb

```diff
@@ -92,10 +124,28 @@ def discard_draft(content_id, options = {})
     post_json!(discard_url(content_id), params)
   end
 
+  # FIXME: Add documentation
+  #
+  # @see https://github.com/alphagov/publishing-api/blob/master/doc/publishing-api-syntactic-usage.md#get-v2linkscontent_id
   def get_links(content_id)
     get_json(links_url(content_id))
   end
 
+  # Patch the links of a content item
+  #
+  # @param content_id [UUID]
+  # @param payload [Hash] A "links hash"
```

Comment:
We should cover semantics:

* A links hash is a hash of tag type -> array of links
* The common tag types are: ...
* The array of tags is unordered, we use the "primary/secondary" pattern for x y z tag types
* When you leave out a tag type, existing tags of that type are unchanged
* When you set a tag type to an empty array, it deletes existing tags of that type
* What happens when the content id doesn't exist?